config/routes.rb

```diff
@@ -1,3 +1,4 @@
 Rails.application.routes.draw do
   root "welcome#index"
+  resources :foods
```

Comment:
We're only creating an `index` right now so we should limit to only the index action: `resources :foods, only: :index`